zipkin-storage/elasticsearch/src/test/java/zipkin2/elasticsearch/integration/ElasticsearchStorageExtension.java

```diff
@@ -36,10 +35,16 @@
 
 class ElasticsearchStorageExtension implements BeforeAllCallback, AfterAllCallback {
   static final Logger LOGGER = LoggerFactory.getLogger(ElasticsearchStorageExtension.class);
+
+  static {
+    // Gives better context when there's an exception such as AbortedStreamException
+    System.setProperty("com.linecorp.armeria.verboseExceptions", "always");
```

Comment:
this has so far not helped, but maybe it will here?

Reply:
Dunno if this static block runs early enough for Armeria to pick it up, armeria reads the property only once at initialization

Reply:
yeah I was hoping to not rely on build config, so will move it up farther if this one doesn't work :P

Reply:
@minwoox @ikhoon @trustin ps any way we can do this without system properties?

Reply:
I think using system properties is the only to change the value. 🤔

Reply:
I think the easy verification is to add a debug statement and see if Armeria already logged it's flags-related stuff, if so too late. If not, it should work :)

Agree with the sentiment of not mucking with build, but for five lines maybe it's still worth not worrying about ordering issues

http://maven.apache.org/surefire/maven-surefire-plugin/examples/system-properties.html
```
<plugin>
        <groupId>org.apache.maven.plugins</groupId>
        <artifactId>maven-surefire-plugin</artifactId>
        <version>3.0.0-M5</version>
        <configuration>
          <systemPropertyVariables>
            <propertyName>propertyValue</propertyName>
```

Reply:
it is also that I often run in IDE and it wouldn't catch those lines, but then again it never fails in the IDE..

Reply:
5eec6f3e6